extension/tamper.css

```diff
@@ -31,6 +31,7 @@ h1 {
 label {
   display: block;
   padding: 5px 0;
+  color: blue;
```

Comment:
cool, if it doesn't work maybe an easier solution is to make the background a constant color? maybe grey?